tests/e2e-tests/specs/activate-and-setup/setup-wizard.test.js

```diff
@@ -60,148 +61,12 @@ describe( 'Store owner can go through store Setup Wizard', () => {
-					event.simulated = true;
-					element.dispatchEvent( event );
-				} );
-
-				await page.$eval( 'input[name="shipping_zones[domestic][flat_rate][cost]"]', e => e.setAttribute( 'value', '10.00' ) );
-
-				// Select Flat Rate shipping method for the rest of the world shipping zone
-				await page.evaluate( () => {
-					document.querySelector( 'select[name="shipping_zones[intl][method]"] > option:nth-child(1)' ).selected = true;
-					let element = document.querySelector( 'select[name="shipping_zones[intl][method]"]' );
-					let event = new Event( 'change', { bubbles: true } );
-					event.simulated = true;
-					element.dispatchEvent( event );
-				} );
-
-				await page.$eval( 'input[name="shipping_zones[intl][flat_rate][cost]"]', e => e.setAttribute( 'value', '20.00' ) );
-
-				// Select product weight and product dimensions options
-				await expect( page ).toSelect( 'select[name="weight_unit"]', 'Pounds' );
-				await expect( page ).toSelect( 'select[name="dimension_unit"]', 'Inches' );
-
-				await Promise.all( [
-					// Click on "Continue" button to move to the next step
-					page.click( 'button[name=save_step]', { text: 'Continue' } ),
-
-					// Wait for the Recommended section to load
-					page.waitForNavigation( { waitUntil: 'networkidle0' } ),
-				] );
-
-				// Fill out recommended section details
-				// Turn off Storefront Theme option
-				// await page.waitForSelector( '#wc_recommended_storefront_theme', { visible: true } );
-				// await page.$eval( '#wc_recommended_storefront_theme', elem => elem.click() );
-
-				// Turn off Automated Taxes option
-				await page.waitForSelector( '#wc_recommended_automated_taxes', { visible: true } );
-				await page.$eval( '#wc_recommended_automated_taxes', elem => elem.click() );
-
-				// Turn off WooCommerce Admin option
-				await page.waitForSelector( '#wc_recommended_wc_admin', { visible: true } );
-				await page.$eval( '#wc_recommended_wc_admin', elem => elem.click() );
-
-				// Turn off Mailchimp option
-				await page.waitForSelector( '#wc_recommended_mailchimp', { visible: true } );
-				await page.$eval( '#wc_recommended_mailchimp', elem => elem.click() );
-
-				// Turn off Facebook option
-				await page.waitForSelector( '#wc_recommended_facebook', { visible: true } );
-				await page.$eval( '#wc_recommended_facebook', elem => elem.click() );
-
-				await Promise.all( [
-					// Click on "Continue" button to move to the next step
-					page.click( 'button[name=save_step]', { text: 'Continue' } ),
-
-					// Wait for the Jetpack section to load
-					page.waitForNavigation( { waitUntil: 'networkidle0' } ),
-				] );
-
-				// Skip activate Jetpack section
-				// Click on "Skip this step" in order to skip Jetpack installation
-				await page.click( '.wc-setup-footer-links' );
-
-				// Finish Setup Wizard - Ready! section
-				// Visit Dashboard
-				await StoreOwnerFlow.openDashboard();
+				await completeOldSetupWizard();
```

Comment:
I think the `else` branch here is superfluous, it seems to me that the `completeOldSetupWizard` effectively runs in either case after the `if ( newSetupWizardNotice ) {}` block.

Reply:
Thanks for noticing it Peter! You are right. I removed the unnecessary `else` statement.